{"status":"OK","result":{"originalLocale":"en","allowViewHistory":true,"creationTimeSeconds":1504311767,"rating":3,"authorHandle":"wrick","modificationTimeSeconds":1504312757,"id":54245,"title":"\u003cp\u003eScala solutions failing â€” bug in runner\u003c/p\u003e","locale":"en","content":"\u003cdiv class\u003d\"ttypography\"\u003e\u003cp\u003eMy rating has been in a free fall due to Scala solutions not running in Codeforces for the last couple of contests. Please see:\u003c/p\u003e\u003cp\u003e\u003ca href\u003d\"/contest/849/submission/29977952\" title\u003d\"Submission 29977952 by wrick\"\u003e29977952\u003c/a\u003e\u003c/p\u003e\u003cp\u003e\u003ca href\u003d\"/contest/842/submission/29890121\" title\u003d\"Submission 29890121 by wrick\"\u003e29890121\u003c/a\u003e\u003c/p\u003e\u003cp\u003e\u003ca href\u003d\"/contest/842/submission/29874042\" title\u003d\"Submission 29874042 by wrick\"\u003e29874042\u003c/a\u003e\u003c/p\u003e\u003cp\u003e\u003ca href\u003d\"/contest/849/submission/29999130\" title\u003d\"Submission 29999130 by wrick\"\u003e29999130\u003c/a\u003e\u003c/p\u003e\u003cp\u003eThey work fine locally and they randomly fail with this error: \u003c/p\u003e \u003cpre\u003e\u003ccode\u003ejava.lang.NoClassDefFoundError: scala/collection/TraversableOnce$class\n\tat IO.\u0026lt;init\u0026gt;(_849A.scala:188)\n\tat IO.\u0026lt;init\u0026gt;(_849A.scala:189)\n\tat IO.\u0026lt;init\u0026gt;(_849A.scala:190)\n\tat CodeForcesApp$class.main(_849A.scala:16)\n\tat _849A$.main(_849A.scala:1)\n\tat _849A.main(_849A.scala)\nCaused by: java.lang.ClassNotFoundException: scala.collection.TraversableOnce$class\n\tat java.net.URLClassLoader.findClass(URLClassLoader.java:381)\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:424)\n\tat sun.misc.Launcher$AppCla...\n\u003c/code\u003e\u003c/pre\u003e\u003cp\u003eCan CodeForces please rerun my submissions and update the ratings?\u003c/p\u003e\u003c/div\u003e","tags":["scala","bug report"]}}